src/org/sage_one_sample/sageone/SageoneSigner.java

```diff
@@ -0,0 +1,130 @@
+package org.sage_one_sample.sageone;
+
+import java.net.*;
+import java.util.*;
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.UnsupportedEncodingException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+public class SageoneSigner {
+  String method;
```

Comment:
These should all be private. If you need to access them outside of the class you should provide getters and setters for them. If they are just used within the class they can just be private